tests/wpt/metadata/css/cssom/ttwf-cssom-doc-ext-load-count.html.ini

```diff
@@ -1,5 +1,4 @@
-[ttwf-cssom-doc-ext-load-count.html]
-  type: testharness
-  [stylesheet.css should be unloaded and styleSheets.length === 0]
```

Comment:
I remember you said this test will keep failing? besides, is it possible to find the root cause of it? 🙇

Reply:
It can be found, but it was even failing before this pull request. So I think it is best to reference this in new issue. Still if you want we can look into it.

Reply:
> but it was even failing before this pull request

yeah, understood so maybe we should keep it labeled as `FAIL` :)?

> So I think it is best to reference this in new issue. Still if you want we can look into it.

Yes, I want to create a new issue for it but I didn't know the root cause for it :( I think it's still good to find the root cause to see if it's related to `ownerNode` itself :)

Reply:
> yeah, understood so maybe we should keep it labeled as `FAIL` :)?

Sorry if I misunderstood, does this mean making it's expectation as FAIL? If it is the case then I have already done that.

Reply:
Sorry, maybe I didn't explain my thought clearly. 

I mean

(the second test was originally labeled as `FAIL`)

https://github.com/servo/servo/blob/cd579f6746515a59a0abab14941165629e13f86d/tests/wpt/web-platform-tests/css/cssom/ttwf-cssom-doc-ext-load-count.html#L39

and in current code change it's removed and only added the 3rd test as `FAIL`

https://github.com/servo/servo/blob/cd579f6746515a59a0abab14941165629e13f86d/tests/wpt/web-platform-tests/css/cssom/ttwf-cssom-doc-ext-load-count.html#L57

So, I think both of them are `FAIL` now.

Reply:
Actually the 2nd test is passing now, so I removed the FAIL label. And due to this test passing, it was revealed that there was some error previously in the 3rd test.
https://github.com/servo/servo/pull/23096#issuecomment-480615964

Reply:
Ahh, I see. Sorry, I misunderstood your comment yesterday. Then your current change is correct! Thanks!